src/modules/profile/profile.resolver.ts

```diff
@@ -0,0 +1,48 @@
+import { Arg, Args, Mutation, Query, Resolver } from "type-graphql";
+import { Service } from "typedi";
+
+import {
+  CreateProfileInput,
+  UpdateProfileInput,
+} from "../../core/dto/profile.dto.js";
+import { GetUserArgs } from "../../core/dto/user.dto.js";
+import { Profile } from "../../core/entities/profile.entity.js";
+import ProfileService from "./profile.service.js";
+
+@Service()
+@Resolver(() => Profile)
+export class ProfileResolver {
+  constructor(private readonly profileService: ProfileService) {}
+
+  @Query(() => Profile, { nullable: true })
+  async getProfile(@Args() params: GetUserArgs) {
```

Comment:
All of these references to `getUserArgs` are going to throw validation errors. Can you please change them all to `@Arg() userId: number` instead? I think going forward I will not make any types for something that has just one property because it seems to make extra work for us. Thanks! (there are 3 of these in this file)